controllers/storagecluster/cephcluster.go

```diff
@@ -411,10 +411,7 @@ func newCephCluster(sc *ocsv1.StorageCluster, cephImage string, kmsConfigMap *co
 		"app": sc.Name,
 	}
 
-	maxLogSize, err := resource.ParseQuantity("500Mi")
-	if err != nil {
-		return &rookCephv1.CephCluster{}, fmt.Errorf("Failed to parse maxLogSize for log rotate. %v", err)
-	}
+	maxLogSize := resource.MustParse("500Mi")
```

Comment:
Can you also remove the error part from the func signature, It is not used anywhere else.

Reply:
Okay done